3m3kalionel/worklist
# #167686037 User signup client

#### What does this PR do?
* It implements user sign up on the client
#### Description of Task to be completed?
* Provide a form for a user to create an account with the following fields
   `Username`, `Telephone`, `Email`, `Password`
#### How should this be manually tested?
* Go to the home page of the app, click on the sign up tab and enter your details. If successful, you should be routed to the dashboard.
#### Any background context you want to provide?
* None
#### What are the relevant pivotal tracker stories?
* [167686037](https://www.pivotaltracker.com/story/show/167686037)
#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/24870657/62789692-69eb6600-bac1-11e9-8918-73e903f3d3da.png)

#### Questions:
* None
